clients/drcachesim/analyzer_multi.cpp

```diff
@@ -521,9 +530,11 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::init_dynamic_schedule()
         sched_ops.deps = sched_type_t::DEPENDENCY_TIMESTAMPS;
     } else if (!op_cpu_schedule_file.get_value().empty()) {
         cpu_schedule_zip_.reset(new zipfile_istream_t(op_cpu_schedule_file.get_value()));
-        sched_ops.mapping = sched_type_t::MAP_TO_RECORDED_OUTPUT;
-        sched_ops.deps = sched_type_t::DEPENDENCY_TIMESTAMPS;
         sched_ops.replay_as_traced_istream = cpu_schedule_zip_.get();
+        if (op_skip_to_timestamp.get_value() == 0) {
```

Comment:
Skipping to a certain timestamp seems to be a different thing than the mapping and deps (wouldn't they behave the same after we've done the skip-to-timestamp?). Explain in comment why we need this condition.

Reply:
Added:

        // -cpu_schedule_file is used for two different things: actually replaying,
        // and just input for -skip_to_timestamp.  Only if -skip_to_timestamp is 0
        // do we actually replay.